src/pcapi/core/categories/subcategories.py

```diff
@@ -1053,6 +1053,76 @@ def __post_init__(self):
     reimbursement_rule=ReimbursementRuleChoices.STANDARD.value,
 )
 # endregion
+# region TECHNICAL
+ACTIVATION_EVENT = Subcategory(
+    id="ACTIVATION_EVENT",
+    category_id=categories.TECHNIQUE.id,
+    matching_type="EventType.ACTIVATION",
+    pro_label="Catégorie technique d'événement d'activation ",
+    app_label="Catégorie technique d'événement d'activation ",
+    search_group=SearchGroupChoices.MATERIEL.value,
```

Comment:
```suggestion
    search_group=SearchGroupChoices.NONE.value,
```

cf https://github.com/pass-culture/pass-culture-api/blob/0caab9622e7d900e0e210fca18316fe4aecb3779/src/pcapi/core/offers/models.py#L485

En ajoutant   `NONE = None` à `SearchGroupChoices`

Reply:
:+1: